src/swarm/node/model/NeoNode.d

```diff
@@ -870,7 +870,8 @@ public class NodeBase ( ConnHandler : ISwarmConnectionHandler ) : INodeBase
     ***************************************************************************/
 
     private void handleUpdateCredentials ( cstring args,
-        void delegate ( cstring response ) send_response )
+        void delegate ( cstring response ) send_response,
+        void delegate ( ref mstring response ) wait_reply )
```

Comment:
@nemanja-boric-sociomantic do we need to change anything more than this here?

Reply:
No, I think that's it. It just enables you to call `wait_reply` from the handler delegate. No more work is needed.